### PR TITLE
[zh-cn] Resync localization strings

### PR DESCRIPTION
**Pull request by mengjiao-liu**
When I read page https://kubernetes.io/zh-cn/docs/setup/production-environment/tools/kubeadm/install-kubeadm/
, I find that the note [legacy_repos_message] is not synchronized.
image

So I resync the file `data/i18n/zh-cn/zh-cn.toml`.

See the [preview page](https://deploy-preview-47244--kubernetes-io-main-staging.netlify.app/zh-cn/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#installing-kubeadm-kubelet-and-kubectl).

### CODE REVIEW

File: data/i18n/zh-cn/zh-cn.toml [hunk 1]
```diff
@@ -10,12 +10,24 @@ other = "(自动生成的页面)"
...
+other = "案例分析："
+
 [caution]
 other = "注意："
 
 [cleanup_heading]
 other = "清理现场"
 
+[china_icp_license]
+other = "ICP license:"
```
Comment (tengqm) [inline data/i18n/zh-cn/zh-cn.toml:29]:
```suggestion
other = "ICP 许可："
```

Comment (mengjiao-liu) [reply data/i18n/zh-cn/zh-cn.toml]: Thanks! Done.

**Comment by windsonsea** [comment]
Thanks.
/lgtm

**Pull request merged**